### PR TITLE
docs: change README to explain the situation

### PR DESCRIPTION
**Pull request by maiste**
Quick change in the README to reflect the new way of deploying and generating the executables.

### CODE REVIEW

**Review by moyodiallo — CHANGES_REQUESTED** [review]
Great, just some suggestions.

File: README.md [hunk 1]
```diff
@@ -1,28 +1,69 @@
...
+```shell
+$ cd sandworm/
+$ opam install . --deps-only --with-dev-setup
+```
+
+Please note that `--with-dev-setup` should only be used for a development purpose.
+
+### Configure
+
+The configuration is in [./sandworm/bin/config.ml](./sandworm/bin/config.ml) file. When running in the pipeline, the _Sandworm_ binary is generated before the execution. As a result, the path taken is the root of this repository. If you want to run it from the [./sandworm](./sandworm) directory, you have to update the path using `../[path]`.
```
Comment (moyodiallo) [inline README.md:30]:
```suggestion
The configuration is in [sandworm/bin/config.ml](./sandworm/bin/config.ml) file. When running in the pipeline, the _sandworm_ binary is generated. Make sure the files artifacts and rclone.conf are in the current directory where _sandworm_ binary is executed.
```

File: README.md [hunk 2]
```diff
@@ -1,28 +1,69 @@
...
+```ocaml
+let s3_bucket_ref = "dune-binary-distribution:/path/to/your/server/dir"
+```
+
+> [!TIP]
+> For our use case, the _RClone_ configuration works with SFTP but, it is compatible with any _RClone_ provider.
+
+## Running
+
+Now your setup is ready, supposing you are in the [./sandworm](./sandworm) directory, you can simply run the program with:
```
Comment (moyodiallo) [inline README.md:54]:
```suggestion
Now your setup is ready, you can simply run the program with:
```

File: README.md [hunk 3]
```diff
@@ -1,28 +1,69 @@
...
+
+```shell
+$ dune exec -- sandworm
```
Comment (moyodiallo) [inline README.md:57]:
```suggestion
$ cd sandworm
$ ls
artifacts rclone.conf
$ dune exec -- sandworm
```

File: README.md [hunk 4]
```diff
@@ -1,28 +1,69 @@
...
+```
+
+or you can build it and execute it:
+
+```shell
+$ dune build
```
Comment (moyodiallo) [inline README.md:63]:
```suggestion
$ cd sandworm
$ dune build
$ ls
artifacts rclone.conf
```

**Review by moyodiallo — CHANGES_REQUESTED** [review]
Great, just some suggestions.

**Comment by maiste** [comment]
@moyodiallo thanks for the review! I have updated the files using most of your suggestions.

**Review by moyodiallo — APPROVED** [review]

**Pull request merged**